src/GuzzleHttp/Middleware.php

```diff
@@ -55,6 +55,8 @@ public static function cache(StorageAdapterInterface $adapter, $debug = false)
                         function (ResponseInterface $response) use ($adapter, $request, $debug) {
                             $adapter->save($request, $response);
 
+                            $response->getBody()->seek(0);
```

Comment:
We can use this function  https://github.com/guzzle/psr7/blob/master/src/functions.php#L250
> The body of the message will only be rewound if a call to `tell()` returns a value other than `0`.